### PR TITLE
Update links and description in Readme.md and Package.md

### PR DESCRIPTION
**Pull request by ayeshLK**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,17 +6,15 @@ Ballerina WebSub Library
...
   [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-The WebSub library is one of the standard library modules of the<a target="_blank" href="https://ballerina.io/"> Ballerina</a> language.
+The WebSub package is one of the standard library packages of the <a target="_blank" href="https://ballerina.io/">Ballerina</a> language.
 
-This library provides the capability to easily introduce publishers and subscribers that are WebSub-compliant. The ballerina/websub library comes with an in-built WebSub Hub service, which can be brought up by publishers that need to bring up a hub.
-
-For example demonstrations of the usage, go to [Ballerina By Examples](https://ballerina.io/swan-lake/learn/by-example/websub-internal-hub-sample.html).
+This package provides the capability to easily introduce publishers and subscribers that are WebSub-compliant. The ballerina/websub package comes with an in-built WebSub Hub service, which can be brought up by publishers that need to bring up a hub.
```
Comment (ThisaruGuruge) [inline README.md:11]: Is this statement correct? Does this package comes with an in-built websubhub service or is it in a separate package (websubhub) ?

File: README.md [hunk 2]
```diff
@@ -6,17 +6,15 @@ Ballerina WebSub Library
...
```
Comment (ThisaruGuruge) [inline README.md:12]: Shall we add the API doc and BBE links here?

Comment (ayeshLK) [reply README.md:12]: * @ThisaruGuruge  currently we do not have BBE's for this module. But we could include API docs here.

File: README.md (hunk 1 above)
Comment (ayeshLK) [reply README.md]: * Updated the document properly.

**Review by ThisaruGuruge — APPROVED** [review]

**Pull request merged**